### PR TITLE
Alphabetize proposals

### PR DESCRIPTION
**Pull request by bterlson**

### CODE REVIEW

File: stage-0-proposals.md [hunk 1]
```diff
@@ -7,24 +7,24 @@ Stage 0 proposals are either
...
-|   | [Method parameter decorators](https://goo.gl/r1XT9b)                                                                                     | Igor Minar                      | 0     |
 |   | [Function expression decorators](https://goo.gl/8MmCMG)                                                                                  | Igor Minar                      | 0     |
-|   | [Zones](https://github.com/domenic/zones) ([spec](https://domenic.github.io/zones/))                                                     | Domenic Denicola & Miško Hevery | 0     |
-|   | [Updates to Tail Calls to include an explicit syntactic opt-in](https://github.com/tc39/proposal-ptc-syntax)                             | Brian Terlson & Eric Faust      | 0     |
-|   | [Nested `import` declarations](https://github.com/tc39/ecma262/pull/646)                                                                 | Ben Newman, Meteor Development Group | 0 |
 |   | [`is{Type}` methods](https://github.com/jasnell/proposal-istypes)                                                                        | James M Snell                   | 0     |
```
Comment (jfbastien) [inline stage-0-proposals.md]: Does backtick alphabetize between "F" and "M"?

File: stage-0-proposals.md [hunk 2]
```diff
@@ -7,24 +7,24 @@ Stage 0 proposals are either
...
+|   | [`String.prototype.at`](https://github.com/mathiasbynens/String.prototype.at)                                                            | Mathias Bynens & Rick Waldron   | 0     |
+|   | [Structured Clone](https://github.com/dslomov-chromium/ecmascript-structured-clone)                                                      | Dmitry Lomov                    | 0     |
+|   | [Updates to Tail Calls to include an explicit syntactic opt-in](https://github.com/tc39/proposal-ptc-syntax)                             | Brian Terlson & Eric Faust      | 0     |
 |   | [WHATWG URL](https://github.com/jasnell/proposal-url)                                                                                    | James M Snell                   | 0     |
+|   | [Zones](https://github.com/domenic/zones) ([spec](https://domenic.github.io/zones/))                                                     | Domenic Denicola & Miško Hevery | 0     |
 |   | [`%contstructor%.construct(thisArg[, arguments])`](https://github.com/jasnell/proposal-construct)                                        | James M Snell                   | 0     |
```
Comment (jfbastien) [inline stage-0-proposals.md]: Percent totally alphabetizes before all the letter! ASCII and all.

**Comment by bterlson** [comment]
@ljharb they are all stage 0, so it's just insertion order. Insertion order makes it hard to scan the list looking for something (say you only vaguely remember the name). Generally, ctrl+f doesn't remove the need for users to be able to find what they're looking for easily.

**Comment by bathos** [comment]
2 cents: I have been using this document frequently and would agree that alphabetization would be useful.

**Comment by ljharb** [comment]
@bterlson I guess i just don't see the benefit, when users who don't know what to ctrl-F won't necessarily be able to intuit the name of the feature anyways. I suppose on the stage 0 list, at least, there's no need to block, but I prefer insertion order, and this seems to just introduce diff churn.

**Comment by bterlson** [comment]
@ljharb FWIW, the usability benefits of sorted lists to enable visual scanning are well understood and documented. I'm also not seeing any argument as to why insertion order is better other than it's currently that way.

**Pull request merged**